modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/StaticHtml2Generator.java

```diff
@@ -215,6 +209,31 @@ private void preparHtmlForGlobalDescription(Swagger swagger) {
         }
     }
 
+    /**
+     * Format to HTML the enums contained in every operations
+     *
+     * @param parameterList The whole parameters contained in one operation
+     * @return String | Html formated enum
+     */
+    public List<CodegenParameter> postProcessParameterEnum(List<CodegenParameter> parameterList) {
+        for(CodegenParameter parameter : parameterList) {
+            if (parameter.isEnum) {
+                parameter.enumFormatted = "";
```

Comment:
Hey, yeah np. I can switch to vendor extensions. am gonna do it today.
ps: thanks for the tips, looks better than what i did !